### PR TITLE
write to log file when rescanning is finished

### PR DESCRIPTION
**Pull request by robust-cuttlefish**
I need it to add bitcoind backend to electrum. Grepping the source, I couldn't find an RPC call to find rescan status to find when rescanning is done. Logging it would be perfect, and only needs 1 line of code.

### CODE REVIEW

**Comment by robust-cuttlefish** [comment]
I need it to add bitcoind backend to electrum. Grepping the source, I couldn't find an RPC call to find rescan status to find when rescanning is done. Logging it would be perfect, and only needs 1 line of code.

**Comment by sipa** [comment]
Please update the PR description.

**Comment by robust-cuttlefish** [comment]
Sorry Pieter, I'm somewhat new to Github and I'm not sure exactly what you meant, but I tried to update the description. Not sure I did it right.

File: src/wallet/rpcdump.cpp [hunk 1]
```diff
@@ -1183,6 +1183,7 @@ UniValue importmulti(const JSONRPCRequest& mainRequest)
             nLowestTimestamp = chainActive.Tip()->GetBlockTime();
         } else {
             fRescan = false;
+            LogPrintf("Rescan completed\n");
```
[the remark this answers is not in the document]
Comment (robust-cuttlefish) [reply src/wallet/rpcdump.cpp]: sorry for wasting your time, I will look where it actually will finish a rescan. ./src/wallet/wallet.cpp, function CBlockIndex looks promising. I was tired.

**Comment by promag** [comment]
@robust-cuttlefish no problem.

If you are calling `importmulti` then you know rescan is finished when you get the response.

**Comment by robust-cuttlefish** [comment]
awesome. that's perfect. many thanks.

**Pull request closed without merging**